src/server/scripts/Northrend/Ulduar/HallsOfStone/brann_bronzebeard.cpp

```diff
@@ -864,14 +1148,9 @@ class iron_golem_custodian : public CreatureScript
     };
 };
 
-class spell_hos_dark_matter_aura : public AuraScript
+class spell_hos_dark_matter : public AuraScript
 {
-    PrepareAuraScript(spell_hos_dark_matter_aura);
-
-    bool Validate(SpellInfo const* /*spellInfo*/) override
-    {
-        return ValidateSpellInfo({ SPELL_DARK_MATTER_H, SPELL_DARK_MATTER });
-    }
```

Comment:
why are you removing the `Validate()` function? Any spell that is beeing casted inside a spell script **must** be validated.

Reply:
I don't remember, it might have been accidental.